### ISSUE TITLE
Feature Request: Tappable note placer

### ISSUE THREAD

**Issue by IceQubed** [Ikeiwa/3D-beatsaber-song-editor#2]
I've got an idea that could make creation of tracks a lot easier and faster - taps. The idea is that you listen to the song while in the song editor, and press a key on the keyboard or tap the mouse button for every note you want to place. These taps are then quantised to fit exact beats (time quantisation is taken from cursor precision). 

Each 'tap' is a placeholder for a beat, which you can then go through afterwards and add a colour/position/angle manually after going through the song tapping. 

These placeholder 'taps' can be deleted, and you can always record more taps by playing the song from any point again.

**Comment by Forstride** [comment]
Definitely think this would really help to streamline song creation.  Although the ability to manually place placeholder beats would be nice too, just for extra precision (Not sure how accurate quantizing them would be).

**Comment by elliotttate** [comment]
This probably should be closed now that it was updated. I've found "bombs" to be good placeholders for now, though it wouldn't hurt to have a more "official" placeholder that just let us toggle the placeholder into a note. The other thing that I'd like to see are allowing the tappable notes to be assign to different keyboard shortcuts. Thanks for all the hard work!!

**Comment by softwaremills** [comment]
True, I tend to put notes in as placeholders, but bombs are even better. Good idea!
